apps/scoutgame/middleware.ts

```diff
@@ -1,17 +1,36 @@
+import { isDraftSeason } from '@packages/dates/utils';
 import { getSession } from '@packages/nextjs/session/getSession';
 import { getPlatform } from '@packages/utils/platform';
 import type { NextRequest } from 'next/server';
 import { NextResponse } from 'next/server';
 
 // These are the links that are only accessible to logged in users
 const privateLinks = ['/profile', '/notifications', '/welcome', '/claim', '/builders-you-know', '/quests', '/accounts'];
+const draftLinks = ['/draft', '/'];
 
 export async function middleware(request: NextRequest) {
   const session = await getSession();
   const isLoggedIn = !!session.scoutId;
   const path = request.nextUrl.pathname;
   const response = NextResponse.next(); // Create a response object to set cookies
   const platform = getPlatform();
+  const draftSeason = isDraftSeason();
+
+  if (draftSeason) {
+    if (!draftLinks.some((link) => path.startsWith(link))) {
+      return NextResponse.redirect(new URL(isLoggedIn ? '/draft' : '/', request.url));
+    }
+
+    if (isLoggedIn) {
+      return NextResponse.redirect(new URL('/draft', request.url));
```

Comment:
i tried running this locally and it just kept sending me to /draft even though i am already on it

Reply:
removed it. thnx